src/SUMMARY.md

```diff
@@ -20,3 +20,5 @@
   - [Initial results](performance-analysis-spec/initial-results.md)
 - [Caveats](caveats/README.md)
 - [Future work](future-work/README.md)
+
+[Version history](version-history/README.md)
```

Comment:
Would this ideally be bulleted as well?

Reply:
It’s not numbered in the Google Doc so I left it unnumbered here

Reply:
I am happy to take credit for that error. But I think we should fix it :-).

Reply:
I believe the reason it's not numbed in the Google Doc is that the heading is not yet an accepted change